source/_docs/email.md

```diff
@@ -126,6 +126,9 @@ This is a common error with the SMTP Authentication Support module. It can be fi
 See [available patch](https://drupal.org/node/1369736#comment-5644064).
 
 ## Frequently Asked Questions
+### Can I access the mail logs for my site?
+No. The mail logs are not accessible.
+
```

Comment:
I'd like to add context for why they are not accessible or move this down the list a bit